book/src/pong-tutorial/pong-tutorial-01.md

```diff
@@ -124,12 +124,13 @@ In `main()` in `main.rs`, we will load the configuration from the file:
 # extern crate amethyst;
 # use amethyst::prelude::*;
 # use amethyst::renderer::DisplayConfig;
-# fn main() {
+# fn main() -> amethyst::Result<()> {
 use amethyst::utils::application_root_dir;
 
-let path = format!("{}/resources/display_config.ron", application_root_dir());
+let path = application_root_dir()?.join("resources/display_config.ron");
```

Comment:
Would be nice to have `application_dir(AsRef<Path>)`

Reply:
Ah, Isn't this technically an API addition? I'd rather do that separately if that's all right by you.

Reply:
Yeah it is, but it can be fixed here. It's a simple function, so I think the friction for another PR is a bit high, but I won't block this.

Once the book section is updated, feel free to r+ this.

Reply:
I've added it now.